### ISSUE TITLE
ASP.NET Core 3 Error

### ISSUE THREAD

**Issue by mattburton** [Moesif/moesif-dotnet#16]
I am attempting to evaluate Moesif for our needs but I'm getting this error under .NET Core 3:
```
System.TypeLoadException: Could not load type 'Microsoft.AspNetCore.Http.Internal.BufferingHelper' from assembly 'Microsoft.AspNetCore.Http, Version=3.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.
   at Moesif.Middleware.NetCore.MoesifMiddlewareNetCore.FormatRequest(HttpRequest request)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Moesif.Middleware.NetCore.MoesifMiddlewareNetCore.FormatRequest(HttpRequest request)
   at Moesif.Middleware.NetCore.MoesifMiddlewareNetCore.Invoke(HttpContext httpContext)
   at Moesif.Middleware.MoesifMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapWhenMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapWhenMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   ...
```
I've seen this error elsewhere in regards to 3.0, such as Sentry, so hopefully that helps:

https://github.com/getsentry/sentry-dotnet/issues/249

Thanks!
Matt

**Comment by dgilling** [comment]
Thanks @mattburton for the detailed info. We'll definitely take a look.

**Comment by mattburton** [comment]
Sure thing - thanks @dgilling

**Comment by dgilling** [comment]
Hi @mattburton We released 1.0.3 which is validated against .NET Core 3.0 and fixes your issue. Thanks for letting us know!

**Comment by mattburton** [comment]
@dgilling thanks for the fast turnaround - most appreciated! I was able to get it up and running with the new version. Also, just a heads up the .NET configuration example in the walkthrough needs a bit of tweaking before someone can copy and paste it directly - instead of this:
```
app.UseMiddleware<MoesifMiddleware>(
    {"ApplicationId", "..."},
    {"LogBody", true}
);
```
I needed to make it an `Dictionary<string, object>` in order to get it working:
```
app.UseMiddleware<MoesifMiddleware>(new Dictionary<string, object>
{
    {"ApplicationId", "..."},
    {"LogBody", true}
});
```
No big deal, just thought I'd let you know. Everything else looking great so far - poking around now - thanks again!
